OracleDatabase/RAC/OracleDNSServer/README.md

```diff
@@ -0,0 +1,67 @@
+# Oracle DNS Server to resolve Oracle RAC IPs
+
+Sample container build files to facilitate installation, configuration, and environment setup for DevOps users.
+
+**IMPORTANT:** This image can be used to setup DNS server for RAC. You can skip this step if you already have a DNS server configure and which can be used for Oracle RAC. You should ensure that the DNS server container is up before starting RAC. This image is provided for test purposes only.
+
+## How to build and run
+You need to make sure that you have at least 350MB of space available for the container to create the files for RAC DNS server.
+
+**IMPORTANT:** If you are behind a proxy, you need to set the `http_proxy or https_proxy` env variable based on your environment before building the image.
+
+The `buildContainerImage.sh` script can assist with building the images. See below for instructions and usage.
+
+The `buildContainerImage.sh` script is a utility shell script that performs MD5 checks and is an easy way to get started. Users can also use the docker build command to build an image with custom configuration parameters. To run the script, go into the `dockerfiles` folder and run the `buildContainerImage.sh` script:
+
+```
+./buildContainerImage.sh-v <Software Version>
+./buildContainerImage.sh -v latest
+```
+NOTE: To build the DNS server image, pass the version latest to `buildContainerImage.sh`. The RAC DNS server image is not tied to any release of the RAC release, you can use `latest` version to build the image.

```

Comment:
`latest` should be wrapped in backticks as it's a literal value not a placeholder.